tools/iso8601.c

```diff
@@ -152,6 +152,7 @@ main(int argc, char **argv)
 
     GError *error = NULL;
     pcmk__output_t *out = NULL;
+    pcmk__output_t *logger_out = NULL;
```

Comment:
Just drop the trace logs. There's no point in duplicating command-line output to logs, because the only way to see the logs is via -VVVVVV on the command line.